changelog/778.feature.rst

```diff
@@ -0,0 +1 @@
+Typehint all Flag classes, which now enables typechecking for Intents(members=True)`` which used to be untyped.
```

Comment:
not entirely sure about the wording here, could use some suggestions

Reply:
```suggestion
Add type hints to all flag constructors, now supporting type-checking for ``Intents(members=True)`` which used to be untyped.
```

Reply:
https://github.com/DisnakeDev/disnake/pull/778/commits/85f082c9c8811d17c2abd0a3a06d4e0c4eebee77